scripts/docker-inner.sh

```diff
@@ -23,22 +24,20 @@ shortname="drush"
 target_dir="$rpm_dir/$fedora_release/drush"
 versions=$(cat $bin/../VERSIONS.txt | grep -v '^#')
 
-for version in $versions; do
-    channel=${version%%.*}
-    name="$shortname-$channel"
+for version in $versions;
```

Comment:
google shell style guide is pretty good and would have you keep the for `for ..; do` on the same line. https://google.github.io/styleguide/shell.xml